test/client/fixtures/sandbox/node/dom-processor-test.js

```diff
@@ -555,3 +555,18 @@ test('xlink:href attribute of svg elements should be overriden (GH-434)(GH-514)'
     strictEqual(nativeMethods.getAttributeNS.call(use, xlinkNameSpaceUrl, 'href'), urlUtils.getProxyUrl(svgUrl));
 });
 
+test('xml:base attribute of svg element should be overriden (GH-477)', function () {
+    var xmlNameSpaceUrl = 'http://www.w3.org/XML/1998/namespace';
+    var svg             = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+    var url             = 'http://domain.com/';
+    var div             = document.createElement('div');
+
+    svg.setAttribute('xml:base', url);
+    strictEqual(nativeMethods.getAttribute.call(svg, 'xml:base'), urlUtils.getProxyUrl(url));
+
+    div.setAttribute('xml:base', url);
+    strictEqual(nativeMethods.getAttribute.call(div, 'xml:base'), url);
+
+    svg.setAttributeNS(xmlNameSpaceUrl, 'xml:base', url);
+    strictEqual(nativeMethods.getAttributeNS.call(svg, xmlNameSpaceUrl, 'base'), urlUtils.getProxyUrl(url));
```

Comment:
Do you mean [this](https://github.com/DevExpress/testcafe-hammerhead/pull/584/files#diff-d5ae870f3bcac870d0359922cccbaecdR96)?